### PR TITLE
New: Tdr 1771 figma plugin json directly from script

### PR DESCRIPTION
**Pull request by wkhan-tapestry**
@aoverstreet-tapestry can you please review PR.

### CODE REVIEW

File: scripts/figma-dt/src/coach-outlet.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        Object.keys(body).forEach((key) => {
+            formatted[key] = {
+                value: helper.getValue(key, body),
+                type: helper.getType(key, body),
+            };
+        });
+    }
+});
+
+module.exports = formatted;
```
Comment (aoverstreet-tapestry) [inline scripts/figma-dt/src/coach-outlet.js:24]: I see this is a duplicated code would it be possible to make a common helper function?

File: scripts/figma-dt/src/coach.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            formatted[key] = body[key]['value'];
+            formatted[key] = {
+                value: helper.getValue(key, body),
+                type: helper.getType(key, body),
+            };
+        });
+    }
+});
+
+module.exports = formatted;
```
Comment (aoverstreet-tapestry) [inline scripts/figma-dt/src/coach.js:25]: Here it is again duplicated.

File: scripts/figma-dt/src/kate-spade.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        Object.keys(body).forEach((key) => {
+            formatted[key] = {
+                value: helper.getValue(key, body),
+                type: helper.getType(key, body),
+            };
+        });
+    }
+});
+
+module.exports = formatted;
```
Comment (aoverstreet-tapestry) [inline scripts/figma-dt/src/kate-spade.js:24]: Here it is again duplicated.

File: scripts/figma-dt/src/stuart-weitzman.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        Object.keys(body).forEach((key) => {
+            formatted[key] = {
+                value: helper.getValue(key, body),
+                type: helper.getType(key, body),
+            };
+        });
+    }
+});
+
+module.exports = formatted;
```
Comment (aoverstreet-tapestry) [inline scripts/figma-dt/src/stuart-weitzman.js:24]: Here it is again duplicated.

File: scripts/figma-dt/app.js [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+app.get('/stuart-weitzman', async function (req, res) {
+    const stuartWeitzmanUtilities = await getSingleUtility('stuart-weitzman');
+    res.send({
+        stuartWeitzman: {
+            ...stuartWeitzmanStyleValues,
+            ...stuartWeitzmanUtilities,
+        },
+    });
+});
+
```
Comment (aoverstreet-tapestry) [inline scripts/figma-dt/app.js:92]: This also seems duplicated and we could probably refactor it a little bit more.

**Comment by p-musi** [comment]
I updated this branch with the latest from the "next" branch, and changed the PR to merge into next.

**Review by p-musi — APPROVED** [review]

**Pull request merged**